yarn.lock

```diff
@@ -6299,6 +6299,11 @@ dompurify@^2.2.3:
   resolved "https://registry.yarnpkg.com/dompurify/-/dompurify-2.2.3.tgz#ec653ba521b39f397c2ca045769438d593ea8a9f"
   integrity sha512-8Hv7Q0FuwD9rWoB6qI2eZsfKbGXfoUVuGHHrE15vgk4ReOKwOkSgbqb2OMFtc0d5besOEkoLkcyuV10zQ2X5gw==
 
+dompurify@^2.2.6:
```

Comment:
seems weird we would then have 2.2.6 and 2.2.3

Reply:
Good catch

Reply:
fixed